store/postgres/src/relational_queries.rs

```diff
@@ -2942,7 +2942,15 @@ fn jsonb_build_object(
 fn iter_column_names<'a, 'b>(
     attribute_names: &'a BTreeSet<String>,
     table: &'b Table,
+    include_block_range_column: bool,
 ) -> impl Iterator<Item = &'b str> {
+    let extra = if include_block_range_column {
+        ["block_range"].iter()
```

Comment:
It would be better to use the constant `crate::block_range::BLOCK_RANGE_COLUMN` here

Reply:
Nice! I didn't notice we already had that.